examples/next/pages/ui/components/storage/storage-browser/default-auth/index.page.tsx

```diff
@@ -15,15 +15,29 @@ import config from './aws-exports';
 
 Amplify.configure(config);
 
-const { StorageBrowser } = createStorageBrowser({
-  actions: {},
-  elements: elementsDefault,
-  config: createAmplifyAuthAdapter({
-    options: { defaultPrefixes: ['public/', 'protected/'] },
-  }),
-});
-
 function Example() {
+  const [identityId, setIdentityId] = React.useState<string>(null);
+
+  useEffect(() => {
+    fetchAuthSession().then((data) => setIdentityId(data.identityId));
+  }, []);
+
+  const { StorageBrowser } = createStorageBrowser({
+    actions: {},
+    elements: elementsDefault,
+    config: createAmplifyAuthAdapter({
+      options: {
+        defaultPrefixes: [
+          'public/',
+          'forbidden/',
```

Comment:
remove I think?

Reply:
My thought here was to have a forbidden folder for testing purposes